homeassistant/components/plugwise/gateway.py

```diff
@@ -77,11 +70,7 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         sw_version=api.smile_version[0],
     )
 
-    platforms = PLATFORMS_GATEWAY
-    if coordinator.data.gateway["single_master_thermostat"] is None:
-        platforms = SENSOR_PLATFORMS
```

Comment:
Do we need to clean up that constant too?

Reply:
Good call, removed